src/main.cpp

```diff
@@ -180,7 +180,6 @@ Napi::Value DDWAFContext::run(const Napi::CallbackInfo& info) {
 
   DDWAF_RET_CODE code = ddwaf_run(this->_context, &data,
                                   &result, (uint64_t) timeout);
-  mlog("Run result action %i", result.action);
```

Comment:
You can still log this, just use `code` instead as it contains the same value.

Reply:
good point